### PR TITLE
Add support to configure some parameters in duplicate message detecto…

### PR DESCRIPTION
**Pull request by samhaa01**
…r and DELAYED_TRANSACTION_TIMEOUT.

### CODE REVIEW

File: coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java [hunk 1]
```diff
@@ -37,6 +37,9 @@
  */
 public abstract class CoapServerBuilder<T extends CoapServerBuilder> {
     private static final int DEFAULT_MAX_DUPLICATION_LIST_SIZE = 10000;
+    private static final int DEFAULT_DUPLICATE_DETECTOR_CLEAN_INTERVAL = 10000;
```
Comment (tuuket01) [inline coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java:40]: Should this have _MILLIS at the end too?

Comment (samhaa01) [reply coap-core/src/main/java/com/mbed/coap/server/CoapServerBuilder.java]: Good catch! I fixed that.

**Review by tuuket01 — CHANGES_REQUESTED** [review]

**Comment by samhaa01** [comment]
fixed that too.

File: coap-core/src/main/java/com/mbed/coap/server/internal/DuplicationDetector.java [hunk 2]
```diff
@@ -34,16 +34,18 @@
  * @author szymon
  */
 public class DuplicationDetector implements Runnable {
-
+    public static final long DEFAULT_DUPLICATION_TIMEOUT = 30000; // Duplicate detection will end for a message after 30 seconds by default.
```
[the remark this answers is not in the document]
Comment (samhaa01) [reply coap-core/src/main/java/com/mbed/coap/server/internal/DuplicationDetector.java]: fixed.

**Review by tuuket01 — APPROVED** [review]

**Review by tuutep01 — APPROVED** [review]

**Pull request merged**